packages/react-sdk/src/components/PresentBar/PresentBar.tsx

```diff
@@ -56,6 +58,23 @@ export function PresentBar() {
   const isPresenting = state.type === 'presenting';
   const isPresentingInEditMode = isPresenting && state.isEditMode;
 
+  const togglePresentationMode = useCallback(() => {
+    const storedGridStatus = localStorage.getItem('showGridState');
+    if (!isPresenting) {
+      if (isShowGrid) {
+        localStorage.setItem('showGridState', 'true');
```

Comment:
We should also unset the value here, if the grid is off. When you leave the board without stopping the presentation with the button, then the old value is still in local-storage:

- turn grid on
- press presentation button (start presentation)
- leave the room
- go back to room
- turn grid off
- press presentation button (start presentation)
- press presentation button (end presentation)

Expected: No grid

What happens: grid